tensorflow_datasets/image_classification/inaturalist.py

```diff
@@ -78,16 +78,11 @@ def _info(self):
   def _split_generators(self, dl_manager):
     output_files = dl_manager.download_and_extract({
         "trainval_images":
-            tfds.download.Resource(
-                url=urllib.parse.urljoin(_URL, "train_val_images.tar.gz"),
-                extract_method=tfds.download.ExtractMethod.NO_EXTRACT),
```

Comment:
Are you sure removing `tfds.download.Resource` is working ? I would expect that the `.tar.gz` is no extracted, so `dl_manager.iter_archive` would crash.

Reply:
@Conchylicultor I did notice that later on. However, the dataset gives a `download error` using the modified code as well as the original code as follows:

`tensorflow_datasets.core.download.downloader.DownloadError: Failed to get url http://people.vision.caltech.edu/~gvanhorn/datasets/inaturalist/fgvc4_competition/test2017.tar.gz. HTTP code: 403.`

Trying to download the dataset manually using the link in the code gives a permission error

`You don't have permission to access /~gvanhorn/datasets/inaturalist/fgvc4_competition/train_val_images.tar.gz on this server.`

I think we would need to find some other alternate place to get the dataset from

Reply:
It means the website has been updated. And the dataset should be updated (but in another PR). We should not merge changes which are not working

Reply:
@Conchylicultor  when we try to visit 
http://people.vision.caltech.edu/~gvanhorn/datasets/inaturalist/fgvc4_competition/
then we have 3 files there but not (train_val_images.tar.gz)
so exactly what should be done?

Reply:
@Conchylicultor Yes, I think we should leave the `inaturalist` dataset and merge the other two

Reply:
It's depends, are those datasets working  with those changes ?